denys/Lesson01-Favorite-Toys/T01.01-Exercise-CreateLayout/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.example.android.favoritetoys">
+
+    <application
+        android:allowBackup="true"
```

Comment:
In case others are wondering, `true` is the default value for `allowBackup`, but the linter seems to want a the value to be set explicitly as there are potential security issues with this setting. No 100 about the implications, but for our own apps I think we ended up just setting `allowBackup` to `false.